src/admin-toolkit-ui/index.tsx

```diff
@@ -334,7 +344,7 @@ export async function initializeAdminData(
     sdkHelpers?.syncEntity?.(
       state.adminToolkitUiEntity,
       [VideoControlState.componentId, TextAnnouncements.componentId],
-      ADMIN_TOOLS_ENTITY,
+      getNextEnumEntityId(engine),
```

Comment:
wait, this ADMIN_TOOLS_ENTITY has to be the same for all the users, so thats why we had the `ADMIN_TOOLS_ENTITY` here

Reply:
you're right! updated